gallery/gallery-war/src/main/java/org/silverpeas/components/gallery/web/GalleryResource.java

```diff
@@ -221,44 +221,18 @@ public Response getSoundContent(@PathParam("soundId") final String soundId) {
   @GET
   @Path(GALLERY_STREAMINGS_PART + "/{streamingId}/" + GALLERY_STREAMING_PROVIDER_DATA_PART)
   @Produces(MediaType.APPLICATION_JSON)
-  public StreamingProviderDataEntity getStreamingProviderDataFromStreamingId(
+  public Response getStreamingProviderDataFromStreamingId(
```

Comment:
Is this web service can be replaced by the one provided by web-core about the streaming?